docs/community_extensions.rst

```diff
@@ -5,21 +5,56 @@ Community Extensions
 
 There are a number of extensions that are not part of the core library but that you may find useful when developing your own nameko services:
 
-    * `nameko-sqlalchemy <http://pypi.python.org/pypi/nameko-sqlalchemy>`_
+    * `nameko-sqlalchemy <https://github.com/onefinestay/nameko-sqlalchemy>`_
 
         A ``DependencyProvider`` for writing to databases with SQLAlchemy. Requires a pure-python or otherwise eventlet-compatible database driver.
 
-    * nameko-memcached (coming soon)
+        Consider combining it with `SQLAlchemy-filters <https://github.com/Overseas-Student-Living/sqlalchemy-filters>`_ to add filtering, sorting and pagination of SQLAlchemy query objects when exposing them over a REST API.
```

Comment:
Strictly I think sqlalchemy-filters is a JSON-serializer for sqlalchemy query operations. It's not tied to HTTP or REST APIs specifially